android/app/src/main/java/org/fossasia/openevent/adapters/SpeakersListAdapter.java

```diff
@@ -76,6 +76,7 @@ protected void publishResults(CharSequence constraint, FilterResults results) {
     public SpeakersListAdapter(List<Speaker> speakers, Activity activity) {
         super(speakers);
         this.activity = activity;
+        this.picassoCache = new PicassoCache(activity.getApplicationContext());
```

Comment:
You can initialize PicassoCache in Application class and then directly use static instance of it